oracle/oracle-how-to/deploy-oke-nodes-using-ubuntu-images.rst

```diff
@@ -0,0 +1,250 @@
+Deploy OKE Nodes using Ubuntu Images
+====================================
+
+Ubuntu images are available for worker nodes on Oracle Kubernetes Engine (OKE) in Oracle Cloud. Currently there are only a select number of suites and Kubernetes versions supported due to this being a Limited Availability release. 
+
+Available Releases
+------------------
+
+.. list-table::
+   :header-rows: 1
+
+   * - Ubuntu Release
+     - OKE Version
+     - Location
+   * - 22.04 (Jammy Jellyfish)
+     - 1.29
+     - `Link <https://intcanonical.objectstorage.us-phoenix-1.oci.customer-oci.com/p/vpqQtYASl8IooEZ_sxfKDnzUkF1b-3lQmXPC_rXf4zARQYoW7ncE8BGGxNqdUuGa/n/intcanonical/b/oke-shared/o/>`_
+
+
+Networking Plugin Availability
+------------------------------
+
+.. list-table::
+   :header-rows: 1
+
+   * - Node Type
+     - Plugin
+     - Supported
+   * - Managed
+     - Flannel
+     - Yes
+   * - 
+     - VCN Native
+     - Yes
+   * - Self-Managed
+     - Flannel
+     - Yes
+   * - 
+     - VCN Native
+     - No
+
+
+Prerequisites
+-------------
+
+You'll need:
+
+- Oracle Cloud compartment to create the nodes.
+
+- Configured and running OKE cluster on Oracle Cloud.
+
+- Oracle's ``oci`` CLI installed.
+
+- ``kubectl`` installed (Self-Managed only).
+
+
+Download an Ubuntu image
```

Comment:
Although this is not technically wrong, the whole download/upload can by bypassed by pointing directly to the image in our storage bucket. I think this is significantly nicer. e.g.

```
oci compute image import from-object-url https://intcanonical.objectstorage.us-phoenix-1.oci.customer-oci.com/p/vpqQtYASl8IooEZ_sxfKDnzUkF1b-3lQmXPC_rXf4zARQYoW7ncE8BGGxNqdUuGa/n/intcanonical/b/oke-shared/o/jammy/oke-1.29/20240820/daily-ubuntu-amd64-minimal-22.04-jammy-v20240820-oke-1.29.img ...
```

Reply:
Good catch.  This significantly reduces the number of steps and the time it takes for a user to import.

Reply:
Very cool, I'll look at merging the 2 sections now